crm_deduplicate_filter/wizards/partner_merge.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class BasePartnerMergeAutomaticWizard(models.TransientModel):
+    _inherit = "base.partner.merge.automatic.wizard"
+
+    exclude_is_company = fields.Boolean("'Is a company?' field selected")
+    exclude_not_parent = fields.Boolean("Parent company not set")
+    exclude_parent = fields.Boolean("Parent company set (Contacts)")
+
+    @api.multi
+    def _process_query(self, query):
+        if any([self.exclude_is_company, self.exclude_not_parent,
```

Comment:
I think this `if` is a bit redundant, since it makes the individual `if`s just below.
I'd do all 3 individual `if`s and after that a single `if filters:`.
It's not a blocker though, since the final result would be the same.